subprojects/core/src/main/java/org/gradle/api/internal/tasks/userinput/DefaultUserInputHandler.java

```diff
@@ -136,6 +142,10 @@ public T transform(String sanitizedInput) {
 
     @Nullable
     private <T> T prompt(String prompt, Transformer<T, String> parser) {
+        if (interrupted.get()) {
+            return null;
+        }
```

Comment:
🤔 is this necessary?  Doesn't this mean that the text from the additional questions won't be sent?

Reply:
Not necessary, but why would we continue sending input text, if we know the input handler got interrupted?